frontend/src/components/Workspace/Visualize/VisualizeItemLayoutMenuIcon.tsx

```diff
@@ -10,13 +10,16 @@ import ListItemText from "@mui/material/ListItemText"
 import Menu from "@mui/material/Menu"
 import MenuItem from "@mui/material/MenuItem"
 
+import { cancelRoiApi } from "api/outputs/Outputs"
```

Comment:
@sanglevinh 
(Not an error in this section, but...)
"Selecting caiman's "rois" from the "Select Roi" pull-down appears to be causing the following system error.

````
ERROR
Network Error
AxiosError: Network Error
    at XMLHttpRequest.handleError (http://localhost:3001/static/js/bundle.js:472033:14)
```

Reply:
@itutu-tienday I have contacted BE to check on this issue

Reply:
@itutu-tienday  Currently, only the cell_roi.json file can be edited. When call api cancel caiman's server will return a 404 status response (Previously it was status 500 and server corrected to 404). I think it's no problem.

Reply:
@sanglevinh 
Thanks.
However, with the following modification, 400 seems to be responded, but if 400 is responded when "Select Roi" is selected, the following system error seems to occur in FE. I would appreciate it if you could handle the error.
- https://github.com/arayabrain/barebone-studio/commit/dd0ef48efa0a64682883c6b00759f74f22a93ee5

- FE error
  ```
  Uncaught runtime errors:
  
  ERROR
  Request failed with status code 400
  AxiosError: Request failed with status code 400
      at settle (http://localhost:3001/static/js/bundle.js:473148:12)
      at XMLHttpRequest.onloadend (http://localhost:3001/static/js/bundle.js:471833:66)
  ```

Reply:
@itutu-tienday yes, i will add catch for this case, pls check it again

Reply:
@sanglevinh 
- Is the error dialog shown above only in this section? (Could it occur in other places as well?)
- We think the above operation is somewhat inappropriate, since it can occur in normal normal normal system operations.
  - Why is this error surely occurring? (Can't the error be suppressed?)

Reply:
@sanglevinh 
When I run the version that you just pushed (11f32a) and select ROI or fluorescence in "Select Item", I seem to get the following error.
→ Is this happening in your environment?

![image](https://github.com/arayabrain/barebone-studio/assets/122338895/273e9d41-ce77-42eb-a7ae-42791582cc7f)

Reply:
@itutu-tienday I have fixed this problem, please help me check again

Reply:
@sanglevinh 
Thanks, seems to be working fine!